docs/getting-started-guides/ubuntu-calico.md

```diff
@@ -58,13 +58,23 @@ On each Node:
 3. All machines should have connectivity to each other and the internet.
 4. This demo assumes that none of the hosts have been configured with any Kubernetes or Calico software.
 
+## Generate TLS Assets
+
+This guide will set up a secure, TLS authenticated API server. Before starting Kubernetes services, you will need to generate TLS Assets for secure client and worker authentication. To get started with OpenSSL certificates, we suggest the CoreOS [guide for generating Kubernetes TLS Assets](https://coreos.com/kubernetes/docs/latest/openssl.html).
+
 ## Setup Master
 
-Download the `calico-kubernetes` repository, which contains the necessary configuration for this guide.
+### Place TLS Assets
+The master requires three certificates, `ca.pem`, `apiserver.pem`, and `apiserver-key.pem`. Send them to your master host and to move them to the `/etc/kubernetes/ssl` folder with the appropriate permissions.
 
 ```
-wget https://github.com/projectcalico/calico-kubernetes/archive/master.tar.gz
-tar -xvf master.tar.gz
+sudo mv ca.pem /etc/kubernetes/ssl/
+sudo mv apiserver.pem /etc/kubernetes/ssl/
+sudo mv apiserver-key.pem /etc/kubernetes/ssl/
+
+# Set Permissions
+sudo chmod 600 /etc/kubernetes/ssl/*-key.pem
```

Comment:
Why are you using a wildcard here? It only matches one file so seems dangerous...